src/app/main/service/localstorage/local-storage.service.ts

```diff
@@ -133,6 +133,18 @@ export class LocalStorageService {
     return localStorage.getItem('UbsOrderId') === 'undefined' ? false : JSON.parse(localStorage.getItem('UbsOrderId'));
   }
 
+  public setUbsFondyOrderId(orderId: string) {
+    localStorage.setItem('UbsFondyOrderId', JSON.stringify(orderId));
+  }
+
+  public getUbsFondyOrderId(): any {
+    return localStorage.getItem('UbsFondyOrderId') === 'undefined' ? false : JSON.parse(localStorage.getItem('UbsFondyOrderId'));
```

Comment:
1. If this method called `getUbsFondyOrderId` I will expect it to return only IDs (`number` or `string`), not a `boolean` value
So if you want to indicate that there is no Order ID you should return just `0`, not `false`.
2. Why do you do `JSON.parse()` here? If this `UbsFondyOrderId` is a number you should parse it with `parseInt()`, if it's a string you should do any additional moves to parse it. Just get it
3. In this method you do `localStorage.getItem('UbsFondyOrderId')` two times if it's not `undefined`
As you do the same thing two times you can improve the performance a bit here
Just save that value at the beginning and then do whatever you want with it

Reply:
1. Fix this issue. but if I use (number | string) I got errors in some components
2. Change code as You ask
3. fix this. (Thanks for advice)